.gitignore

```diff
@@ -0,0 +1,2 @@
+.swiftpm
+Package.resolved
```

Comment:
Since the only dependency here is the package plugin, it makes sense to me to ignore the Package.resolved file, to ensure that we're always using the latest version of the plugin.

You might also want to consider ignoring the `/.build` folder, see https://github.com/apple/swift-markdown/blob/main/.gitignore for inspiration.

Reply:
Changed in a4c4c43c.